cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -502,6 +503,40 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_node_token_lookup_index() {
+    profileQuery(
+      title = "Node TOKEN LOOKUP index example",
```

Comment:
Opinion:

We have been a bit inconsistent with the naming of these but [here](https://neo4j.com/docs/operations-manual/current/performance/index-configuration/#index-configuration-token-lookup-use-and-significance) we refer to them as "node label lookup indexes" whereas [here](https://neo4j.com/docs/upgrade-migration-guide/current/migration/index-upgrade/#index-migration-token-lookup-indexes) we call them "node lookup index".

I think the "label" would be helpful and then we could leave out the token. WDYT?
```suggestion
      title = "Node label LOOKUP index example",
```